Assets/Scripts/KeyLocationController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityStandardAssets.Characters.FirstPerson;
+
+
+public class KeyLocationController : MonoBehaviour
+{
+    private float radius;
+    public GameObject sphere;
+    public GameObject[] players = new GameObject[2];
+    private float speedMod;
+
+    void Start()
+    {
+        //players = GameObject.FindGameObjectsWithTag("Player");
+        radius = (sphere.transform.localScale.x/2)-1; //-1 to reduce screen clipping with sphere
+    }
+
+    void Update()
+    {
+        players = GameObject.FindGameObjectsWithTag("Player"); //Locate in start/singlerun instead. Idk with Photon stuff
+
```

Comment:
So we have to find players on every update? We can delete the commented one in Start I guess